packages/amplify_authenticator/lib/src/mixins/authenticator_text_field.dart

```diff
@@ -26,7 +26,7 @@ mixin AuthenticatorTextField<FieldType,
   Widget buildFormField(BuildContext context) {
     final inputResolver = stringResolver.inputs;
     final hintText = widget.hintText == null
-        ? inputResolver.resolve(context, widget.hintTextKey!)
+        ? widget.hintTextKey?.resolve(context, inputResolver)
```

Comment:
Not directly related to the PR, but I happened to try to add a custom field without hint text, and this threw.

Reply:
Which form field type? I think hintText is required in `static SignUpFormField custom`... maybe we should do that elsewhere?

Reply:
I think all of the other form fields have it as optional, although none of them were really being used until this PR. We should make it the same. I would be fine with either. Since hintText is optional on `TextFormField`, I would lean towards making it optional on our form fields.

I can try to remember to update this as well in the follow up